homeassistant/components/climate/zwave.py

```diff
@@ -269,9 +270,10 @@ def set_swing_mode(self, swing_mode):
     @property
     def device_state_attributes(self):
         """Return the device specific state attributes."""
-        data = {}
+        data = zwave.ZWaveDeviceEntity(self._value,
```

Comment:
This is very expensive as you are instantiating a new object for each invocation. Instead, just use inheritance:

```python
data = super().device_state_attributes
```